<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::MaterialTab</class>
 <widget class="QWidget" name="GammaRay::MaterialTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>362</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="GammaRay::DeferredTreeView" name="materialPropertyView">
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="GammaRay::DeferredTreeView" name="shaderList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GammaRay::CodeEditor" name="shaderEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::DeferredTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">ui/deferredtreeview.h</header>
  </customwidget>
  <customwidget>
   <class>GammaRay::CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">ui/codeeditor/codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
